pkg/files.go

```diff
@@ -370,3 +370,16 @@ func sshFilesByPattern(pattern string, config *SSHConfig) ([]string, error) {
 	filePaths := buf.String()
 	return strings.Split(strings.TrimSpace(filePaths), "\n"), nil
 }
+
+func UniqueFileInfos(fileInfos []FileInfo) []FileInfo {
```

Comment:
What about slice.CompactFunc here

https://pkg.go.dev/slices#CompactFunc

Reply:
modified and tested on both previous and new method